twoliter/embedded/rpm2img

```diff
@@ -240,6 +206,9 @@ INVENTORY_DATA="$(jq --arg PKG_PREFIX "bottlerocket-" \
 INVENTORY_DATA="$(jq --slurp 'sort_by(.Name)' <<< "${INVENTORY_DATA}" | jq '{"Content": .}')"
 printf "%s\n" "${INVENTORY_DATA}" > "${ROOT_MOUNT}/usr/share/bottlerocket/application-inventory.json"
 
+# install CA certificates
+install_ca_certs
```

Comment:
This should take the arguments it needs, rather than relying on implicit global state:
```suggestion
install_ca_certs "${ROOT_MOUNT}"
```

`partyplanner` is a good example of how to approach this extraction - the end result should be less coupled to the original file.